Gem/AWS/MetricsCDK/aws_metrics/real_time_data_processing.py

```diff
@@ -0,0 +1,297 @@
+        self._stack = stack
+        self._input_stream_arn = input_stream_arn
+        self._application_name = application_name
+
+        self._create_analytics_processing_lambda()
+
+        self._create_analytics_application()
+
+    def _create_analytics_application(self) -> None:
+        """
+        Generate the Kinesis data analytics application to process the real-time data.
+        The sample application filters input events and counts the total number of login within one minute.
+
+        :return: The created Kinesis data analytics application.
+        """
+        self._analytics_application_role = self._create_analytics_application_role()
+
+        self._analytics_application = analytics.CfnApplication(
+            self._stack,
+            'AnalyticsApplication',
+            application_name=resource_name_sanitizer.sanitize_resource_name(
+                f'{self._stack.stack_name}-AnalyticsApplication', 'kinesis_application'),
+            inputs=[
+                analytics.CfnApplication.InputProperty(
+                    input_schema=analytics.CfnApplication.InputSchemaProperty(
+                        record_columns=[
+                            analytics.CfnApplication.RecordColumnProperty(
+                                name='event_id',
+                                sql_type='VARCHAR(32)',
+                                mapping='$.event_id'
+                            ),
+                            analytics.CfnApplication.RecordColumnProperty(
+                                name='event_type',
+                                sql_type='VARCHAR(32)',
+                                mapping='$.event_type'
+                            ),
+                            analytics.CfnApplication.RecordColumnProperty(
+                                name='event_name',
+                                sql_type='VARCHAR(32)',
+                                mapping='$.event_name'
+                            ),
+                            analytics.CfnApplication.RecordColumnProperty(
+                                name='event_version',
+                                sql_type='VARCHAR(32)',
+                                mapping='$.event_version'
+                            ),
+                            analytics.CfnApplication.RecordColumnProperty(
+                                name='event_timestamp',
+                                sql_type='TIMESTAMP',
+                                mapping='$.event_timestamp'
+                            ),
+                            analytics.CfnApplication.RecordColumnProperty(
+                                name='application_id',
+                                sql_type='VARCHAR(32)',
+                                mapping='$.application_id'
+                            ),
+                            analytics.CfnApplication.RecordColumnProperty(
+                                name='client_connection_count',
```

Comment:
this column property has been added but is not called out in your [change list](https://github.com/aws-lumberyard-dev/o3de-multiplayersample/pull/2#issue-1289412432). can you please list it as well and maybe do a pass to see if any other changes weren't called out? (Ideally the c/p from the AWS Metrics gem and the changes made to the logic would be in separate commits so it'd be easer to see what changed)

Reply:
Just created a draft PR to diff against the existing AWSMetrics CDK. This PR should include all the modifications and feel like it could help with the review.